packages/core/src/shared/__test__/rectangle.spec.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { describe, expect, it } from 'vitest';
+
+import { Rectangle } from '../rectangle';
+
+describe('test rectangle', () => {
```

Comment:
```suggestion
describe('test "Rectangle"', () => {
```

Please use double quotes to surrond the API you're about to test.